modules/bim/app/assets/stylesheets/bim/ifc_viewer/generic.sass

```diff
@@ -1,6 +1,7 @@
 // -------------------------- GENERIC --------------------------
 @import "openproject/variables"
 @import "openproject/mixins"
+@import "openproject/generic"
```

Comment:
Why did you import the generic file here? It should not be necessary, as the all styles are loaded in the BIM module, too. Thus when applying the `-error` class, it should already style the resizer accordingly.

Reply:
Hi @HDinger ,

Thank you for your feedback.

If I remove this line it doesn't work for me. I realized that the variables, mixins, and other styles used in the BIM project were imported here. I imported them and it worked so I thought this was the way to go.

Do you have any idea about why it's not working on my machine without importing the generic partials here?

Reply:
Mhhh that is weird. For comparison, this is how the compiled code looks for me (without importing the generic file):

<img width="1525" alt="Bildschirmfoto 2020-04-28 um 09 02 24" src="https://user-images.githubusercontent.com/7457313/80457247-47d79380-892f-11ea-868e-894c5193d8bf.png">

Could you please check, how it looks for you? I assume, that some of the styles are not correctly loaded, that is why you need to import the generic.sass and still see the resizer in red.

Reply:
It seems that importing the generic.scss gives it precedence over the openproject.scss
![Captura de pantalla 2020-04-28 a las 9 28 11](https://user-images.githubusercontent.com/25689432/80460366-0695b280-8934-11ea-985b-75d53ce7b4df.png).
Adding !important works, thanks.

Reply:
Ah okay, that makes sense..